textile/chat-features.textile

```diff
@@ -1029,8 +959,4 @@ For non-chat-specific codes, the status code for the error should align with the
   // Room was released before the operation could complete.
   // To be accompanied with status code 400.
   RoomReleasedBeforeOperationCompleted = 102106
-
-  // The room operation failed because the room was in an invalid state
-  // This error may be accompanied with status code 400 or 500, depending on the source of the error.
-  RoomInInvalidState = 102107
```

Comment:
This error is still referred to in the spec.

Reply:
Hold the thought on that one - I've opened a Slack thread on what we do around CHA-RL11 and tagged you :)